src/main/java/mate/academy/spring/mapper/impl/response/OrderResponseMapper.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.spring.mapper.impl.response;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponseMapper implements DtoResponseMapper<OrderResponseDto, Order> {
+    @Override
+    public OrderResponseDto toDto(Order order) {
+        OrderResponseDto orderResponseDto = new OrderResponseDto();
+        orderResponseDto.setId(order.getId());
+        orderResponseDto.setUser(order.getUser());
+        orderResponseDto.setTickets(order.getTickets());
```

Comment:
```suggestion
        orderResponseDto.setTicketsId(object.getTickets()
                .stream()
                .map(Ticket::getId)
                .collect(Collectors.toList()));
```
we should set tickets Id , cause in OrderResponse we should declare List<Long> ticketsId, but not List<Ticket> tickets.